srcpkgs/stumpwm/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'stumpwm'
+pkgname=stumpwm
+version=22.11
+revision=1
+build_style=gnu-configure
+hostmakedepends="sbcl base-devel"
+makedepends="common-lisp-ppcre common-lisp-alexandria common-lisp-clx"
+depends="common-lisp-ppcre common-lisp-alexandria common-lisp-clx"
+short_desc="Stump Window Manager"
+maintainer="ctoid <funk443@yandex.com>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/stumpwm/stumpwm"
+distfiles="https://github.com/stumpwm/stumpwm/archive/refs/tags/${version}.tar.gz"
+checksum="36940f3b2c1ba54219a8d3a784103cfa83f84c652d1dbd7d3b7360c55e5ef026"
+nopie=yes
+nostrip=yes
+nocross="SBCL seems won't be able to find libraries while cross-compiling"
```

Comment:
More of lack of cross compilation by sbcl in general.

Reply:
What do you mean?